blocks/table/table.js

```diff
@@ -13,6 +13,46 @@ function buildCell(col, rowNum) {
   return cell;
 }
 
+function buildWeatherCell(col, rowNum) {
+  const cell = rowNum > 0 ? document.createElement('td') : document.createElement('th');
+  const cellContainer = document.createElement('div');
+  if (rowNum) {
+    // eslint-disable-next-line eqeqeq
+    if (cell.textContent == parseInt(cell.textContent, 10)) {
+      // if cell contents are only numerical
+      cell.classList.add('table-cell-num');
+    }
+  }
+  if (cell.nodeName === 'TD') {
+    cellContainer.classList.add('weather-cell');
+  }
+
+  switch (rowNum) {
+    case 1:
+      cellContainer.innerHTML = '<img src="/icons/inclement-weather-clear.svg">';
+      break;
+    case 2:
+      cellContainer.innerHTML = '<img src="/icons/inclement-weather-aware.svg">';
+      break;
+    case 3:
+      cellContainer.innerHTML = '<img src="/icons/inclement-weather-prepared.svg">';
+      break;
+    case 4:
+      cellContainer.innerHTML = '<img src="/icons/inclement-weather-action.svg">';
+      break;
+    case 5:
+      cellContainer.innerHTML = '<img src="/icons/inclement-weather-closure.svg">';
+      break;
+    default: // Do Nothing
+  }
```

Comment:
is there a specific reason that these are added in code based on row number rather than authored in the correct row?

Reply:
The ask was to forcibly apply these logos and specific background colours to the first column (rows 1-5).

The authoring team indicated that these logos and colours will at no point change and only text be authorable.